bigger body fonts (fixes #78)

ngokevin opened the pull request:
# After

<img width="637" alt="screen shot 2016-05-31 at 11 29 33 am" src="https://cloud.githubusercontent.com/assets/674727/15686256/4c0707fe-2724-11e6-8ee2-dee039c5c111.png">

# Before

<img width="654" alt="screen shot 2016-05-27 at 11 24 54 am" src="https://cloud.githubusercontent.com/assets/674727/15617404/aec761b0-23fd-11e6-9d61-2933d4fb6648.png">

cvan commented:
I personally like it, but I worry about the lightness and thinness of the font here lending to poor legibility and contrast.

ngokevin commented:
Okay. I'll try just making the body font bigger, and play with headings light weight.

ngokevin commented:
Updated to simply just increasing some font-sizes.

- Including code font size (increased size, decreased padding)
- Including table font size by very slightly
- Add more padding to table header row, removed extraneous bottom border from last row

cvan commented:
feel free to make the sidebar larger (so there isn't as much wrapping):

![image](https://cloud.githubusercontent.com/assets/203725/15686334/a1de592a-2724-11e6-928a-941870b3addc.png)

![image](https://cloud.githubusercontent.com/assets/203725/15686341/a800340e-2724-11e6-994d-ab8768f5595f.png)

cvan commented:
![image](https://cloud.githubusercontent.com/assets/203725/15686811/bca8d3be-2726-11e6-8f81-9eadef606e46.png)

![image](https://cloud.githubusercontent.com/assets/203725/15686795/b18fd02c-2726-11e6-81d9-5496e5b2c6a6.png)

cvan commented on themes/aframe/source/css/_settings.styl:
```diff
@@ -9,11 +9,12 @@ $logo-font = $body-font
-$body-font-weight = normal
+$body-font-size = 1.1rem
+$body-font-weight = 400
 $body-line-height = 1.5
```
can you change this to `1.6`?

and can you add this to `_common.styl`?
```styl
.copy li + li
    margin-top: .35rem
```

ngokevin replied:
k

cvan commented:
r+ looks great to me

ngokevin commented:
Thanks!
